docs/reference/configuration.adoc

```diff
@@ -67,6 +67,22 @@ To use a property on the command line, pass it as a command-line parameter to th
 $ java -Dswarm.bind.address=127.0.0.1 -jar myapp-swarm.jar
 ----
 
+[discrete]
```

Comment:
There are several issues with the content, I'll send a pull request against `kenfinnigan:SWARM-1790` today (Mar 15), please do not merge yet if possible.

Reply:
Damn, thought I'd done well to document! Oh well, let me know where I messed up for next time